frontend/src/lib/utils/answers.ts

```diff
@@ -136,3 +149,37 @@ export const DATE_FORMATS: Record<DateType, Intl.DateTimeFormatOptions> = {
     weekday: 'long'
   }
 };
+
+/**
+ * Check wheter an answer to a question in empty.
+ * @param question The Question object
+ * @param answer The Candidate's answer
+ * @returns A boolean value indicating whether the answer is empty
+ */
+export function answerIsEmpty(question: QuestionProps, answer: AnswerProps): boolean {
```

Comment:
It seems we might be passing in nullish `answer`s so maybe change to `answer?: AnswerProps` and add as the first line inside the function:

```ts
if (!answer) return true;
```

Then we can remove the whole `if (answer)` check.